docs/how_tos/theming.md

```diff
@@ -0,0 +1,204 @@
+# Theming support with `@edx/paragon` and `@edx/brand`
```

Comment:
we need to refactor `@edx/` to `@openedx/` in docs, as since December all npm packages are published in the scope of `@openedx/`

Reply:
@brian-smith-tcril I couldn't see frontend-platform among npm published packages, shouldn't it be published with @openedx scope as well?
https://www.npmjs.com/search?q=%40openedx

Reply:
I think that one is still only published to the `@edx` scope. I agree it would be good to make the switch but it hasn't made it to the top of the priority list yet.